tests/integrations/subprocess/test_Subprocess__check_output.py

```diff
@@ -40,6 +42,52 @@ def test_call_with_path_arg(mock_sub, capsys, tmp_path):
     assert capsys.readouterr().out == ""
 
 
+@pytest.mark.parametrize("platform", ["linux", "darwin", "win32"])
+@pytest.mark.parametrize("start_new_session", [None, True, False])
+def test_call_with_start_new_session(mock_sub, capsys, platform, start_new_session):
+    "start_new_session is passed thru on Linux and macOS but converted for Windows"
+
+    mock_sub.command.sys.platform = platform
+    mock_sub.check_output(['hello', 'world'], start_new_session=start_new_session)
+
+    if platform == "win32":
+        mock_sub._subprocess.check_output.assert_called_with(
+            ['hello', 'world'],
+            text=True,
+            **({'creationflags': 0x8000200} if start_new_session is True else {}),
```

Comment:
As above; the original parametrize can be simplified, and this argument can be made part of the parametrize.

Reply:
Added to `parametrize`.